stable/metallb/README.md

```diff
@@ -87,6 +87,8 @@ By default, this chart does not install a configuration for MetalLB, and simply
 warns you that you must follow [the configuration instructions on MetalLB's
 website][metallb-config] to create an appropriate ConfigMap.
 
+**Please note:** By default, this chart expects a ConfigMap named 'metallb-config' within the same namespace as the chart is deployed. _This is different than the MetalLB documentation_, which asks you to create a ConfigMap in the `metallb-config` namespace, with the name of 'config'.
+
```

Comment:
If I look at the docs, linked in the issue, the namespae is `metallb-system` not `metallb-config`. Can you please update or explain the difference?

Reply:
Updated - that was a mistake on my part. See recent commit